plugins/lookup/tss.py

```diff
@@ -26,6 +26,13 @@
         description: The integer ID of the secret.
         required: true
         type: int
+    fetch_secret_ids_from_folder:
+        description:
+            - Boolean flag which indicates whether secret ids are in a folder is fetched by folder ID or not.
+            - If true then _terms value will be considered as a folder ID else secret ID.
```

Comment:
```suggestion
            - If V(true) then the terms will be considered as a folder IDs. Otherwise (default), they are considered as secret IDs.
```